static/bulma.css

```diff
@@ -0,0 +1,10719 @@
+/*! bulma.io v0.6.2 | MIT License | github.com/jgthms/bulma */
```

Comment:
No `bulm.min.css` inside the project ?

Reply:
Nop il y en a pas dans le package
![screenshot_20180228_174306](https://user-images.githubusercontent.com/8822421/36800044-d6321f14-1cae-11e8-84b2-c99f8f26a925.png)

Reply:
I compressed it manually.